package.json

```diff
@@ -29,12 +29,14 @@
     "draft-js-import-html": "^1.4.1",
     "draftjs-filters": "^2.3.0",
     "history": "^4.10.1",
+    "moment": "^2.24.0",
```

Comment:
These two packages come with porting over the logic used in Gutenberg.

Reply:
I seem to remember that moment was used as it was better at translations. But I am not sure if I am miss remembering this.

Reply:
We can reconsider this after alpha, as we'll have to replace the GB date picker anyway.

Reply:
Yep, however, I'm thinking that perhaps we could remove/replace this when adjusting the calendar in general post-alpha.

Will look into `date-fns` a bit later when the rest of the things have been fixed.